client/components/chart/index.jsx

```diff
@@ -92,17 +94,17 @@ module.exports = React.createClass( {
 	},
 
 	getData: function() {
-		var data = this.props.data;
+		let data = this.props.data;
 
 		data = data.slice( 0 - this.state.maxBars );
 
 		return data;
 	},
 
 	getValues: function() {
-		var data = this.getData();
+		let data = this.getData();
 
-		data = data.map( function ( item ) {
+		data = data.map( function( item ) {
```

Comment:
nope. if it doesn't refer to `this` then it need not be bound

however, we could benefit here from fat-arrow or helper functions…

also, it's probably a bit misleading to be mangling `data`

```js
return data.map( item => item.value );

// or
import { property } from 'lodash';

return data.map( property( 'value' ) );

// better still, don't create a new closure on every call
const valueProp = property( 'value' );
…
return data.map( valueProp );
```